.github/actions/prepare-workspace/action.yml

```diff
@@ -0,0 +1,63 @@
+name: Prepare Teleport workspace
+description: Prepares Teleport workspace folder
+inputs:
+  workload_identity_provider:
+    description: Google Cloud identity provider
+    required: false
+    default: projects/671314886056/locations/global/workloadIdentityPools/github-pool/providers/github-provider
+
+  service_account:
+    description: Google Cloud Storage service account name
+    required: false
+    default: teleport-storage@just-terminus-366813.iam.gserviceaccount.com
```

Comment:
Same question as above, what is this default service account?

Reply:
Same, you should create it on the Google side